xlators/cluster/ec/src/ec.c

```diff
@@ -1794,7 +1794,7 @@ struct volume_options options[] = {
      .type = GF_OPTION_TYPE_INT,
      .min = 1,
      .max = 1024,
-     .default_value = "1",
+     .default_value = "32",
```

Comment:
I don't think this change is related to this PR. Please update it.

Reply:
I think some other commit was also pushed along with this..